internal/provider/resource_consul_cluster.go

```diff
@@ -150,76 +150,81 @@ func resourceConsulCluster() *schema.Resource {
 				Computed:    true,
 			},
 			// computed outputs
-			"organization_id": {
-				Description: "The ID of the organization this HCP Consul cluster is located in.",
+			"cloud_provider": {
+				Description: "The provider where the HCP Consul cluster is located.",
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
-			"project_id": {
-				Description: "The ID of the project this HCP Consul cluster is located in.",
+			"cluster_state": {
```

Comment:
that's the main change. But there are two other behavior changes. Highlights from description:

- add `state`
- remove DELETED Consul clusters from state (not FAILED)
- set Consul client config resource properties after setting the rest of the cluster  resource properties (ie set state after each API call, rather than at the end of calling all of them)